docs/content/documentation/templates/overview.md

```diff
@@ -339,6 +339,18 @@ as below.
 {{ response }}
 ```
 
+When no other parameters are specified the URL will always be retrieved using a HTTP GET request.
+Using the parameter `method`, since version 0.14.0, you can also choose to retrieve the URL using a POST request.
+
+When using `method="POST"` you can also use the parameters `body` and `contenttype`.
+The parameter body is the actual contents sent in the POST request.
+The parameter contenttype should be the mimetype of the body.
+
+```jinja2
+{% set postresponse = load_data(url="https://postman-echo.com/post", format="json", method="POST", body="{i_am:'json'}", contenttype="application/json") %}
```

Comment:
That's not valid JSON for the body. Can we get some real example?

Reply:
i commited a new example calling the kroki.io service to generate an SVG